scripts/validate_docstrings.py

```diff
@@ -1021,7 +400,7 @@ def header(title, width=80, char="#"):
         choices=format_opts,
         help="format of the output when validating "
         "multiple docstrings (ignored when validating one)."
-        f"It can be {str(format_opts)[1:-1]}",
+        "It can be {}".format(str(format_opts)[1:-1]),
```

Comment:
I think worth keeping the f-strings

Reply:
Yep, missed that in a merge, which are quite tricky. Thanks.